src/components/Explorer/Base/Result/RunLookup/RunLookup.tsx

```diff
@@ -32,10 +44,21 @@ export const RunLookup = ({ runId, filters }: Props) => {
     React.useEffect(() => {
         if (!runId) return
         getRunTitle(runId).then(setPageTitle)
+        callApi(runId).then((data) => {
+            setTableData(data)
+        })
+        return () => {
+            setTableData([])
+        }
     }, [runId])
 
     const LinkButtons = context.result.LinkButtons
 
+    async function callApi(runId: string) {
+        const result = await fetch(`https://api.serratus.io/palmprint/run=${runId}`)
+        return result.json()
+    }
+
     const instructions = (
         <div className='text-center'>Click a family heatmap to view sequence-level matches</div>
```

Comment:
This feels a little out of place now, but probably nothing to worry about for this PR.